index.js

```diff
@@ -80,7 +80,7 @@ function updateTKK() {
         if (Number(window.TKK.split('.')[0]) === now) {
             resolve();
         } else {
-            got('https://translate.google.com').then(function (res) {
+            got('https://translate.google.'+opts.tld).then(function (res) {
```

Comment:
We should pass `opts` via exported function `get` to be able to use `opts.tld` here.

Reply:
Not exactly sure how I'd be able to do that, though. Do you have any ideas as to how I should go about implementing this?

Reply:
Resolved in your implementation.